Source/Thirdweb/Private/ThirdwebWalletHandle.cpp

```diff
@@ -62,7 +63,8 @@ bool FWalletHandle::CreateInAppEmailWallet(const FString& Email, FWalletHandle&
 			nullptr
 		).AssignResult(Error))
 		{
-			Wallet = FWalletHandle(FWalletHandle::InApp, Error);
+			Wallet = FWalletHandle(InApp, Error);
+			FThirdwebAnalytics::SendConnectEvent(Wallet.ToAddress(), Wallet.GetTypeString());
```

Comment:
Since we need wallet address will have to wait for it to be actually connected, and it's not always connected upon creation, unlike privatekeywallet/smartwallet